benchmarks/hdg/CMakeLists.txt

```diff
@@ -25,176 +25,164 @@
-
-        FEELPP_ENVIRONMENT_WITH_OPTIONS( makeAbout(), makeOptions() );
-
-        BOOST_AUTO_TEST_SUITE( ${modulename} )
-
-        BOOST_AUTO_TEST_CASE( ${BENCH} )
-        //int main( int argc, char** argv )
-        {
-          //Feel::Assert::setLog( \"${BENCH}.assert\");
-          //Feel::Environment env( argc, argv, makeOptions() );
-          Hdg<${dim}, ${orderP}> hdg;
-          hdg.convergence();
-        }
-        BOOST_AUTO_TEST_SUITE_END()
-        \n" )
-
-      feelpp_add_application(${targetname} SRCS ${BENCH}.cpp DEFS FEELPP_${dim}D USE_FACE_MESH )
-  endforeach() # order
-endforeach() # dim
-
-###################################################
-
-# Linear elasticity
-#foreach( geom 1 2 )
-foreach( dim 2 3)
-  foreach( orderP 0 1 2 3 4)
-    set(modulename "hdg_linear_elasticity")
-    set(BENCH "${modulename}_${dim}D_P${orderP}" )
-    set(targetname ${BENCH})
-    FILE(WRITE ${CMAKE_CURRENT_BINARY_DIR}/${BENCH}.cpp
-      "// this file is auto-generated by Feel++ \n\n
-    #define BOOST_TEST_MODULE hdg_linear_elasticity
-    #include <testsuite/testsuite.hpp>
-    #include<hdg_linear_elasticity.hpp>\n\n
-    using namespace Feel;
-
-    FEELPP_ENVIRONMENT_WITH_OPTIONS( makeAbout(), makeOptions() );
-
-    BOOST_AUTO_TEST_SUITE( hdg_linear_elasticity )
-
-    BOOST_AUTO_TEST_CASE( hdg_linear_elasticity_${dim}D_P${order} )
-    //int main( int argc, char** argv )
-    {
-      //Feel::Assert::setLog( \"${BENCH}.assert\");
-      //Feel::Environment env( argc, argv, makeOptions() );
-      Hdg<${dim}, ${orderP}> hdg;
-      hdg.run();
-    }
-    BOOST_AUTO_TEST_SUITE_END()
-    \n" )
-
-    feelpp_add_application(${targetname} SRCS ${BENCH}.cpp DEFS FEELPP_${dim}D )
-  endforeach() # order
-endforeach() # dim
-# endforeach() # geom
-
-# Linear elasticity - IBC
-#foreach( geom 1 2 )
-foreach( dim 2 3)# 3
-  foreach( orderP 0 1 2 3 4)
-    set(modulename "hdg_linear_elasticity_ibc")
-    set(BENCH "${modulename}_${dim}D_P${orderP}" )
-    set(targetname ${BENCH})
-    FILE(WRITE ${CMAKE_CURRENT_BINARY_DIR}/${BENCH}.cpp
-      "// this file is auto-generated by Feel++ \n\n
-    #define BOOST_TEST_MODULE hdg_linear_elasticity
-    #include <testsuite/testsuite.hpp>
-    #include<hdg_linear_elasticity_ibc.hpp>\n\n
-    using namespace Feel;
-
-    FEELPP_ENVIRONMENT_WITH_OPTIONS( makeAbout(), makeOptions() );
-
-    BOOST_AUTO_TEST_SUITE( hdg_linear_elasticity_ibc )
-
-    BOOST_AUTO_TEST_CASE( hdg_linear_elasticity_ibc_${dim}D_P${order} )
-    //int main( int argc, char** argv )
-    {
-      //Feel::Assert::setLog( \"${BENCH}.assert\");
-      //Feel::Environment env( argc, argv, makeOptions() );
-      Hdg<${dim}, ${orderP}> hdg;
-      hdg.run();
-    }
-    BOOST_AUTO_TEST_SUITE_END()
-    \n" )
-
-    feelpp_add_application(${targetname} SRCS ${BENCH}.cpp DEFS FEELPP_${dim}D )
-  endforeach() # order
-endforeach() # dim
-# endforeach() # geom
+####################################################
```

Comment:
This code is not using the toolboxes classes but it is not updated since we are not directly using it. 
For the moment I kept it with the comment to decide what to do